images/cilium/.dockerignore

```diff
@@ -1,33 +0,0 @@
-# Copyright Authors of Cilium
-# SPDX-License-Identifier: Apache-2.0
-
-# since this is image is built with root context, most of the files
-# need to be excluded for faster builds and to avoid spoiling build
-# cache due to unchecked files (like configs or random binaries)
-*
-
-# must-have toplevel files
```

Comment:
I hope we're not adding any extra files due to these being ignored... Before we remove these can we confirm that besides them being ignored there isn't any unexpected files ending up in images?

Reply:
yup makes sense, this only affects the build context and file additions from the local context are generally specified per-file, as well as those directories no longer applying due to being build from within the Dockerfile directory context.